modules/migration-installing-cam-operator-ocp-3.adoc

```diff
@@ -11,7 +11,12 @@ The CAM Operator installs Restic and Velero.
 .Prerequisites
 
 * You must have `podman` installed.
-* You must have access to the link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry].
+* Your {product-title} 3 cluster must be configured to pull images from `registry.redhat.io`.
++
+[NOTE]
+====
+The `regsitry.redhat.io` registry does not allow unauthenticated access. You must have `registry.redhat.io` configured with credentials in order to authenticate. See this link:https://access.redhat.com/solutions/3772061[Red Hat Solution] for instructions. Be sure to distribute the credentials to all nodes in your cluster.
```

Comment:
Do we need to emphasize the user who is using old OCP v3 clusters (3.7-3.10) must execute this solution?
> Older OCP v3 clusters (3.7-3.10) must be configured to use the new registry.redhat.io in order to pull CAM images, OCP v3.11 is excluded from this since it was the first release to support the new registry. Customers attempting to deploy CAM on older OCP v3 without proper registry setup will encounter pull failures.